### PR TITLE
Tweaks to text (mostly capitalisation)

### PR DESCRIPTION
**Pull request by poll-busily**
Touches page titles, some table headers, menu items. There's some noise in the diffs because my editor was trimming whitespace - sorry about that.

### CODE REVIEW

**Comment by magicbug** [comment]
Accepting this PR might take a while due to the sheer number of changes made see https://github.com/magicbug/Cloudlog/wiki/Development-Notes

Due to previous commits like this causing breaking changes each controller changed will need checked

**Comment by poll-busily** [comment]
No worries. I'll tidy it up a bit and push another revision.

**Comment by poll-busily** [comment]
Ahh, that's better. Apologies for the formatting changes before - SNR should be much better now. There are still controller changes but you'll see they're minor.

File: application/views/contesting/index.php [hunk 1]
```diff
@@ -379,7 +379,7 @@
                                         </div>
                                 </div>
                             <button type="reset" class="btn btn-sm btn-warning" onclick="reset_log_fields()">Reset</button>
-                            <button type="button" class="btn btn-sm btn-primary" onclick="logQso();"><i class="fas fa-save"></i> Log QSO</button>
```
Comment (poll-busily) [inline application/views/contesting/index.php:382]: "Save QSO" was used elsewhere so I'm aiming to standardise on that.

File: application/views/notes/edit.php [hunk 2]
```diff
@@ -47,7 +47,7 @@
 	</div>
 
 	<input type="hidden" name="id" value="<?php echo $id; ?>" />
-	<button type="submit" value="Submit" class="btn btn-primary">Save</button>
```
Comment (poll-busily) [inline application/views/notes/edit.php:50]: Many save buttons are in the form "Save X", where X is a QSO or whatever. Using the same format for notes.

File: application/views/contesting/index.php (hunk 1 above)
Comment (dg9vh) [reply application/views/contesting/index.php:382]: I don't think that saving a QSO is the same as logging a QSO... here we are in Contest-Context creating new log-entries, with "save" you are in edtiting context...

Comment (poll-busily) [reply application/views/contesting/index.php:382]: Good point. Let me go back and review the current usage again.

Comment (AndreasK79) [reply application/views/contesting/index.php:382]: I feel that "Log QSO" is more correct, but you are also saving the QSO in the database. I named it "Log QSO" in the Contest part.

Comment (poll-busily) [reply application/views/contesting/index.php:382]: @magicbug, would you like:

- "Save QSO" all around. I'll keep this commit as-is.
- "Log QSO" all around. I'll change the buttons in the Live QSO and Post QSO pages to match the Contest Logging page and I'll undo this part of my commit.

**Pull request merged**